src/cache-utils.ts

```diff
@@ -34,15 +34,19 @@ export const getPackageManagerInfo = async (packageManager: string) => {
 export const getCacheDirectoryPath = async (
   packageManagerInfo: PackageManagerInfo
 ) => {
-  const stdout = await getCommandOutput(
-    packageManagerInfo.getCacheFolderCommand
-  );
+  let pathList: string[] = [];
+
+  for (let command of packageManagerInfo.cacheFolderCommandList) {
+    pathList.push(await getCommandOutput(command));
+  }
```

Comment:
I think it will be better to use this snippet:
```
pathList = packageManagerInfo.cacheFolderCommandList.map(command => getCommandOutput(command));
```

Reply:
Done, review again, please.